### PR TITLE
Macidonian Temple and Wonder

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/mace/temple.xml [hunk 1]
```diff
@@ -7,6 +7,9 @@
   <Identity>
     <Civ>mace</Civ>
     <SpecificName>Naos</SpecificName>
+    <History>
+        	The Macedonians still worshiped and sacrificed to the old Greek gods. However after the conquests of Alexander the Great, the pantheon to worship was significantly expanded. Immigrants and traders brought brought their religions all across the empire. A single city could house places of worship for not only Zeus and Athena, but gods from Egypt, Persia, and even Yahweh, the God of Israel.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/mace/temple.xml:11]:
```suggestion
        	The Macedonians still worshiped and sacrificed to the old Greek gods. However, after the conquests of Alexander the Great, the pantheon to worship was significantly expanded. Immigrants and traders brought their religions all across the empire. A single city could house places of worship for not only Zeus and Athena but gods from Egypt, Persia, and even Yahweh, the God of Israel.
```

File: 0-ad-history-mod-package/simulation/templates/structures/mace/wonder.xml [hunk 2]
```diff
@@ -7,6 +7,9 @@
   <Identity>
     <Civ>mace</Civ>
     <SpecificName>Anaktoron ton Aigai</SpecificName>
+    <History>
+        The Palace of Pella was the home to the Macedonian kings, a sprawling complex covering an area over 90,000 square yards (75,000 square meters). It was constructed by Phillip II, and was Alexander the Great’s birthplace. Alexander expanded the palace as did other Macedonian rulers after him. The city of Pella and the palace grew as a prominent location during the Hellenistic age. Sadly, the Palace was destroyed in 168 BC after the Macedonian were defeated by the roman empire.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/mace/wonder.xml:11]:
```suggestion
        The Palace of Pella was the home to the Macedonian kings, a sprawling complex covering an area of over 90,000 square yards (75,000 square meters). It was constructed by Phillip II and was Alexander the Great’s birthplace. Alexander expanded the palace, as did other Macedonian rulers after him. The city of Pella and the palace grew as a prominent location during the Hellenistic age. Sadly, the palace was destroyed in 168 BC after Macedonia was defeated by the Roman Empire.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this OK to merge?

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**